man/systemd.network.xml

```diff
@@ -2430,9 +2419,38 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
       </varlistentry>
       <varlistentry>
         <term><varname>BindToInterface=</varname></term>
-        <listitem>Takes a boolean value. When <literal>yes</literal>, DHCP server socket will be bound
-        to its network interface and all socket communication will be restricted to this interface.
-        Defaults to <literal>yes</literal>.
+        <listitem>
+          <para>Takes a boolean value. When <literal>yes</literal>, DHCP server socket will be bound
+          to its network interface and all socket communication will be restricted to this interface.
+          Defaults to <literal>yes</literal>.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>RelayTarget=</varname></term>
+        <listitem>
+          <para>Takes an IPv4 address, which must be in the format
+          described in
+          <citerefentry project='man-pages'><refentrytitle>inet_pton</refentrytitle><manvolnum>3</manvolnum></citerefentry>.
+          Turns this DHCP server into a DHCP relay agent. See <ulink url="https://tools.ietf.org/html/rfc1542">RFC 1542</ulink>.
+          The address is the address of DHCP server or another relay agent to forward DHCP messages to and from.
+          Check also <varname>BindToInterface=</varname> option. Turning it off is required for relaying messages outside.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>RelayAgentCircuitId=</varname></term>
+        <listitem>
+          <para>Specifies value for Agent Circuit ID sub-option of Relay Agent Information Option.
+          Takes a string, which must be in the format <literal>string:%s</literal>,
+          where <literal>%s</literal> should be replaced with the value of the suboption.
+          Defaults to unset(no suboption is generated). Ignored if <varname>RelayTarget=</varname> is not specified.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>RelayAgentRemoteId=</varname></term>
+        <listitem>
+          <para>Specifies value for Agent Remote ID sub-option of Relay Agent Information Option.
+          Takes a string, which must be in the format "string:%s", where %s will be a value of the suboption.
+          Defaults to unset(no suboption is generated). Ignored if <varname>RelayTarget=</varname> is not specified.</para>
```

Comment:
space before (

also "no suboption"? you mean no Agent Remote ID suboption? say so then

sometimes you write sub-option, sometimes suboption. Please stick to one spelling